src/nvim/search.c

```diff
@@ -5689,7 +5689,7 @@ void find_pattern_in_path(char_u *ptr, Direction dir, size_t len, bool whole, bo
 
           // we read a line, set "already" to check this "line" later
           // if depth >= 0 we'll increase files[depth].lnum far
-          // bellow  -- Acevedo
+          // below.
```

Comment:
Match <https://github.com/vim/vim/commit/8e7d6223f630690b72b387eaed704bf01f3f29d2>
```suggestion
          // below  -- Acevedo
```